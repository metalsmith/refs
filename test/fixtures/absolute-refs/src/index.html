---
refs:
  about: file:/about/index.html
---