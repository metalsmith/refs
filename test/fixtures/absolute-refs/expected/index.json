{
  "refs": {
    "about": {
      "id": "about/index.html"
    }
  },
  "id": "index.html"
}